### DISCUSSION TITLE
Possible Code Issue in release-8.0.0 of Questionset Editor

### DISCUSSION THREAD

**Discussion by rajnishdargan** [Sunbird-inQuiry/Community#116] [Bugs]
In the Questionset Editor code for release-8.0.0, the API for updating comments was integrated. It performs API calls formatted like this:
```
curl '{{host}}/action/questionset/v2/comment/update/do_is' \
  -H 'Content-Type: application/json' \
  --data-raw '{"request":{"comments":[{"comment":"testing"}]}}'
```
Since release-8.0.0 has not yet been officially released, it appears there may be a code issue within the [release-8.0.0](https://github.com/Sunbird-inQuiry/editor/tree/release-8.0.0) branch of the editor.

You can find the previous discussion regarding this issue [here](https://github.com/orgs/Sunbird-inQuiry/discussions/115).

Action Items:
@kiranharidas187 @ramkumar-pacewisdom, could you please verify this issue and plan to implement a fix if possible?
The changes on the editor side seem to be associated with this ticket: [IQ-125](https://project-sunbird.atlassian.net/browse/IQ-125).

@pallakartheekreddy, can you confirm if any modifications related to the comment update were made in the release-8.0.0 of the inQuiry service? Additionally, please check and fix the [comment update route](https://github.com/Sunbird-inQuiry/inquiry-api-service/blob/release-7.0.0/assessment-api/assessment-service/conf/routes#L78) after confirming.

Thank you for your attention to this matter!

@kiranharidas187 @ramkumar-pacewisdom @Vivek-M-08
@pallakartheekreddy @krgauraw
@coolbung @snehal0904 @itsvick

**Comment by pallakartheekreddy** [comment]
@rajnishdargan We will check and update on this.
